spec/fixtures/css_page.rb

```diff
@@ -8,6 +8,8 @@ class CSSPage < SitePrism::Page
   elements :elements_one, 'a.a b.b'
   elements :elements_two, '.many'
 
+  element :no_such_element, 'a.b c.d'
```

Comment:
This fixture needs adding to both css and xpath pages

Reply:
Do we test the deprecation messages as well ?

Reply:
We can test those yes